src/Appwrite/Platform/Modules/Functions/Workers/Builds.php

```diff
@@ -705,6 +708,33 @@ protected function buildDeployment(Device $deviceForFunctions, Device $deviceFor
                 throw new \Exception('Build size should be less than ' . number_format($buildSizeLimit / 1048576, 2) . ' MBs.');
             }
 
+            if ($resource->getCollection() === 'sites' && empty($resource->getAttribute('adapter'))) {
+                $listFilesCommand = "cd /usr/local/build && cd " . \escapeshellarg($resource->getAttribute('outputDirectory')) . " && find . -name 'node_modules' -prune -o -type f -print";
```

Comment:
Todo to replace this with a properly structured execution command later on.

Reply:
added